specification/attributes/column_naming_convention.md

```diff
@@ -18,16 +18,17 @@ Naming convention for columns appearing in billing data.
 
 ## Requirements
 
-* All columns defined by FOCUS MUST follow the following rules:
+* All columns defined by FOCUS(*) MUST follow the following rules:
   * Column IDs MUST use [Pascal case](https://techterms.com/definition/pascalcase).
   * Column IDs MUST NOT use abbreviations.
-  * Column IDs SHOULD NOT use acronyms.
   * Column IDs MUST be alphanumeric with no special characters.
   * Columns that have an ID and a Name MUST have the `Id` or `Name` suffix in the Column ID. Display Name for a Column MAY avoid the Name suffix if there are no other columns with the same name prefix.
-* Custom (e.g., provider-defined) columns SHOULD follow the same rules as FOCUS columns listed above.
-* Columns that have an ID and a Name MUST have the `Id` or `Name` suffix in the Column ID. Display Name for a Column MAY avoid the `Name` suffix if it is considered superfluous.
-* Columns with the `Category` suffix must be normalized.
+  * Column IDs SHOULD NOT use acronyms.
+  * Column IDs SHOULD NOT exceed 59 characters.
```

Comment:
This length of 59 is very specific for a SHOULD column.  Would prefer a rounder number; suggest 50.  Also suggest a bit of language around the reason why.  I don't feel as strongly as about that part, though.
```suggestion
  * Column IDs SHOULD NOT exceed 50 characters to accommodate column length restrictions of various data repositories.
```